test/test-utils/threads.ts

```diff
@@ -137,6 +137,7 @@ export const mkThread = ({
     const thread = room.createThread(rootEvent.getId(), rootEvent, events, true);
     // So that we do not have to mock the thread loading
     thread.initialEventsFetched = true;
+    thread.addEvents(events, false);
```

Comment:
This seems to cause `EventTile › EventTile thread summary › removes the thread summary when thread is deleted` to fail. I'm not quite sure why.

Reply:
This has been solved by, we should remove that line https://github.com/matrix-org/matrix-js-sdk/pull/2951

Reply:
See also #9742, this was a bad cherry-pick of #9723.